instaloader/instaloader.py

```diff
@@ -1334,6 +1334,9 @@ def check_profile_id(self, profile_name: str, latest_stamps: Optional[LatestStam
                                                                                                   profile_id))
                 profile_from_id = Profile.from_id(self.context, profile_id)
                 newname = profile_from_id.username
+                if profile_name == newname:
+                    self.context.error("Profile {0} could not be retrieved by its name, but by its ID.".format(profile_name))
```

Comment:
```suggestion
                    self.context.error(f"Warning: Profile {profile_name} could not be retrieved by its name, but by its ID.")
```

- prefix the output with "Warning:", since the reported problem does not have any impact on the final result
- new f"" format string